pkg/scheduler/core/division_algorithm.go

```diff
@@ -179,45 +200,45 @@ func divideRemainingReplicas(remainingReplicas int, desiredReplicaInfos map[stri
 	}
 }
 
-func scaleScheduleByReplicaDivisionPreference(spec *workv1alpha2.ResourceBindingSpec, preference policyv1alpha1.ReplicaDivisionPreference,
-	preSelectedClusters []*clusterv1alpha1.Cluster) ([]workv1alpha2.TargetCluster, error) {
-	assignedReplicas := util.GetSumOfReplicas(spec.Clusters)
-	if assignedReplicas > spec.Replicas {
-		newTargetClusters, err := scaleDownScheduleByReplicaDivisionPreference(spec, preference)
-		if err != nil {
-			return nil, fmt.Errorf("failed to scaleDown: %v", err)
-		}
-		return newTargetClusters, nil
-	} else if assignedReplicas < spec.Replicas {
-		newTargetClusters, err := scaleUpScheduleByReplicaDivisionPreference(spec, preSelectedClusters, preference, assignedReplicas)
-		if err != nil {
-			return nil, fmt.Errorf("failed to scaleUp: %v", err)
-		}
-		return newTargetClusters, nil
-	} else {
-		return spec.Clusters, nil
-	}
+func scaleDownScheduleByReplicaDivisionPreference(
+	spec *workv1alpha2.ResourceBindingSpec,
+	preference policyv1alpha1.ReplicaDivisionPreference,
+) ([]workv1alpha2.TargetCluster, error) {
+	// The previous scheduling result will be the weight reference of scaling down.
+	// In other words, we scale down the replicas proportionally by their scheduled replicas.
+	return divideReplicasByPreference(spec.Clusters, spec.Replicas, preference, emptyClusterNames)
```

Comment:
```suggestion
	return divideReplicasByPreference(spec.Clusters, spec.Replicas, preference, sets.NewString())
```
Just a suggestion. And just a feeling that the `emptyClusterNames` should be removed as looks weird.